plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/UIUtils.java

```diff
@@ -1427,6 +1428,27 @@ public static boolean launchProgram(String path)
         return Program.launch(path);
     }
 
+    public static void showInSystemExplorer(@NotNull String path) {
+        showInSystemExplorer(path, getActiveWorkbenchWindow());
+    }
+
+    /* Adapted from org.eclipse.ui.internal.ide.dialogs.ResourceInfoPage */
+    public static void showInSystemExplorer(@NotNull String path, @NotNull IServiceLocator serviceLocator) {
+        final ICommandService commandService = serviceLocator.getService(ICommandService.class);
+        final IHandlerService handlerService = serviceLocator.getService(IHandlerService.class);
+        final Command command = commandService.getCommand(ShowInSystemExplorerHandler.ID);
+
+        if (command.isDefined()) {
+            final Map<String, String> params = Collections.singletonMap(ShowInSystemExplorerHandler.RESOURCE_PATH_PARAMETER, path);
+            final ParameterizedCommand pc = ParameterizedCommand.generateCommand(command, params);
+
+            try {
+                handlerService.executeCommand(pc, null);
+            } catch (Exception e) {
+                DBWorkbench.getPlatformUI().showError("Error showing path in system explorer", "Can't show '" + path + "' in system explorer", e);
+            }
+        }
+    }
```

Comment:
I believe these methods belong to RuntimeUtils. Moreover, it's not exactly clear why we need two distinct methods.